hudi-client/src/main/java/org/apache/hudi/index/HoodieIndex.java

```diff
@@ -118,9 +121,10 @@ protected HoodieIndex(HoodieWriteConfig config) {
   /**
    * Each index type should implement it's own logic to release any resources acquired during the process.
    */
-  public void close() {}
+  public void close() {
+  }
 
   public enum IndexType {
-    HBASE, INMEMORY, BLOOM, GLOBAL_BLOOM
+    HBASE, INMEMORY, BLOOM, GLOBAL_BLOOM, SIMPLE
```

Comment:
We should also implement a GLOBAL_SIMPLE?

Reply:
Sure, will add it in a diff patch.

Reply:
let's fix it in this itself?